internal/ctl/connectors/manage.go

```diff
@@ -82,6 +90,18 @@ if you specify --once then it will sync once and then exit.`,
 	manageCmd.Flags().DurationVar(&params.HTTPClientTimeout, "http-client-timeout", params.HTTPClientTimeout, "HTTP client timeout")
 	_ = viper.BindPFlag("http-client-timeout", manageCmd.PersistentFlags().Lookup("http-client-timeout"))
 
+	manageCmd.Flags().IntVar(&params.GlobalMaxConnectorRestarts, "global-connector-max-restarts", params.GlobalMaxConnectorRestarts, "maximum times a failed connector will be restarted")
```

Comment:
maybe these strings could be `const` as they appear in a couple of places?

Reply:
I don't think that would be an improvement.

Reply:
Im talking about these strings `global-connector-max-restarts` etc... I think it is an improvement to move these into consts.